addons/account/wizard/wizard_tax_adjustments.py

```diff
@@ -11,6 +11,11 @@ class TaxAdjustments(models.TransientModel):
     def _get_default_journal(self):
         return self.env['account.journal'].search([('type', '=', 'general')], limit=1).id
 
+    @api.depends('journal_id')
+    def _compute_fiscal_countries(self):
+        fiscal_position_countries = self.env['account.fiscal.position'].search([('company_id', '=', self.journal_id.company_id.id), ('foreign_vat', '!=', False)]).country_id
+        self.fiscal_countries = self.journal_id.company_id.country_id + fiscal_position_countries
```

Comment:
You need to support computation for multiple records in a loop (`for record in self:`)
The search in a loop is "okay"ish since we are on a transient

Also, break long lines on multiple shorter lines

The declaration of fields is supposed to be the very first thing on a model. The compute methods should go under.
https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#symbols-and-conventions